### ISSUE TITLE
Bump compatibility with GLPI from 10.0.0 to 10.0.1

### ISSUE THREAD

**Issue by ilGino** [fusioninventory/fusioninventory-for-glpi#3341]
Dear all,

I installed GLPI 10.0.1 (security fix patch version).
Then I installed Fusion Inventory for GLPI 10 plugin, but could not enable it because it required GLPI = 10.0.0

So I think it is not compatible with GLPI 10.0.1 due to this check during the Enable process.

Possibly it is just a matter of extending the compatiblity from 10.0.0 to 10.0.1 because I don't think core GLPI changed anything with regards to Fuisninventory plugin for GLPI.

Thank you all for attention and Regards
GG

**Comment by ddurieux** [comment]
In version 9.5.x we had 5 'major versions' and break the plugin, that's why it's filtered for each minor version.
Il will check it.

**Comment by rafacss** [comment]
after migrating to glpi 10.0.1 my fusioninvenory also stopped working.

**Comment by ddurieux** [comment]
Yes, you need waiting the new version on couple days (this week, but not know when exactly.

**Comment by tamasborsstriva** [comment]
> Yes, you need waiting the new version on couple days (this week, but not know when exactly.

Hi,

do you have any estimate day for the fix?

Thanks

**Comment by ddurieux** [comment]
Monday
